portal/src/WizardContentLayout.tsx

```diff
@@ -53,6 +53,7 @@ export function WizardDescription(
 ): React.ReactElement {
   return <Text block={true}>{props.children}</Text>;
 }
+import cn from "classnames";
```

Comment:
This screen show the navigation menu in mobile view. But the navigation has no effect.